Dockerfile

```diff
@@ -48,7 +48,7 @@ FROM alpine
 RUN apk add --no-cache ca-certificates bash curl jq libc6-compat
 
  # Install Azure Cli
-RUN apk add --no-cache --update python3 py3-pip
+RUN apk add --no-cache --update python3 py3-pip go git && git clone https://github.com/portworx/torpedo.git /torpedo-gin
```

Comment:
@gvavinash Do we have check if run is for UI then build gin other not ?

Reply:
Its separate file from the torpedo runs and which will use for various UI projects